data/crac/crac-util/src/main/java/com/farao_community/farao/data/crac_util/CracCleaner.java

```diff
@@ -0,0 +1,62 @@
+/*
+ *  Copyright (c) 2020, RTE (http://www.rte-france.com)
+ *  This Source Code Form is subject to the terms of the Mozilla Public
+ *  License, v. 2.0. If a copy of the MPL was not distributed with this
+ *  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.farao_community.farao.data.crac_util;
+
+import com.farao_community.farao.data.crac_api.Crac;
+import com.powsybl.iidm.network.Identifiable;
+import com.powsybl.iidm.network.Network;
+
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Set;
+import java.util.regex.Pattern;
+
+/**
+ * @author Viktor Terrier {@literal <viktor.terrier at rte-france.com>}
+ */
+
+public final class CracCleaner {
+
+    private CracCleaner() {
+    }
+
+    public static void cracAliasesUtil(Crac crac, Network network) {
+        // List (without duplicates) all the crac elements that need to be found in the network
+        Set<String> elementIds = new HashSet<>();
+        crac.getCnecs().forEach(cnec -> elementIds.add(cnec.getNetworkElement().getId()));
+        crac.getContingencies().forEach(contingency -> contingency.getNetworkElements().forEach(networkElement -> elementIds.add(networkElement.getId())));
+
+        // Try to find a corresponding element in the network, and add elementId as an alias
+        elementIds.forEach(elementId -> {
+            Optional<Identifiable<?>> correspondingElement = network.getIdentifiables().stream().filter(identifiable -> anyMatch(identifiable, elementId)).findAny();
+            correspondingElement.ifPresent(identifiable -> identifiable.addAlias(elementId));
+        });
+    }
+
+    private static boolean anyMatch(Identifiable<?> identifiable, String cnecId) {
+        return nameMatches(identifiable, cnecId, false) ||
+            aliasMatches(identifiable, cnecId, false) ||
+            nameMatches(identifiable, cnecId, true) ||
+            aliasMatches(identifiable, cnecId, true);
```

Comment:
This might be dangerous to do that. We actually decided to avoid doing that kind of things on CORE CC data, as reversing the cnecId without reversing the Threshold direction (direct/opposite/both) change the way your input data are interpreted.

It only works with `direction = both`. 

The problem is that the crac-api does not allow today to know the direction of the threshold (:sweat:). So i would add a big comment here, indicating that it only works for Cnec with `direction = both` and that it corrupts the other Cnec.

And also add a `TODO : only look for the elements with reverse ID if all the thresholds of the cnec have a BOTH direction`.